__mocks__/cookie.js

```diff
@@ -0,0 +1,5 @@
+let cookie = {};
```

Comment:
For this script, do we need to mock?  Can we simply write to `document.cookie` and then `expect` within tests?

Reply:
We don't need to mock.  I was just trying to leave things in place in case people needed to extend it later.  I can remove the mock

Reply:
So it turns out that because getAvailableCookies expects there to be a global doucment.cookie it needs to be there.